keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -165,6 +170,25 @@ Determine the date that a given secret was last used:
 
 If `kube-apiserver-legacy-service-account-token-tracking` is unavailable, no secret would be removed.
 
+Mark the secrets as invalid and recover:
+
+1. The secrets will be added a label with `kubernetes.io/invalid-legacy-token`: true.
+2. If the users use the invalid tokens, in the Validate() function of
+   "kubernetes/pkg/serviceaccount/legacy.go", it will detect the usage of
+   invalid tokens and return the error information, telling the users to
+   re-activate the token by updating the label value or use the tokenrequest. At
+   the same time, the tokens will be updated with the new `kubernetes.io/legacy-token-last-used` date.
+3. If the users don't use the invalid tokens, one year later after the tokens
```

Comment:
use the configured `--legacy-service-account-token-clean-up-period` rather than hard-coding a year

Reply:
Done.